com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -661,6 +661,8 @@ void DrawCameraType()
             EditorGUILayout.PropertyField(m_AdditionalCameraDataCameraTypeProp, Styles.cameraType);
             if (EditorGUI.EndChangeCheck())
             {
+                camera.targetDisplay = (((CameraRenderType)m_AdditionalCameraDataCameraTypeProp.intValue) == CameraRenderType.Base)
```

Comment:
Yea I was thinking about same solution. However this will fail if user will change target display with script.

Reply:
Ohh I didn't think of that case.
Then we would need to override the function for URP specifically since the additional camera data is unique for URP.

Reply:
Yea, but users can still change target display in camera. https://docs.unity3d.com/ScriptReference/Camera-targetDisplay.html

Reply:
I added a statement in universalRenderPipeline setting each camera overlay to -1 for editor only.
Are the penalty to high or okay when it only runs in the editor?